test/e2e_node/topology_manager_test.go

```diff
@@ -203,13 +203,17 @@ func findNUMANodeWithoutSRIOVDevices(configMap *v1.ConfigMap, numaNodes int) (in
 	return findNUMANodeWithoutSRIOVDevicesFromSysfs(numaNodes)
 }
 
-func configureTopologyManagerInKubelet(oldCfg *kubeletconfig.KubeletConfiguration, policy, scope string, configMap *v1.ConfigMap, numaNodes int) (*kubeletconfig.KubeletConfiguration, string) {
+func configureTopologyManagerInKubelet(oldCfg *kubeletconfig.KubeletConfiguration, policy, scope string, topologyOptions map[string]string, configMap *v1.ConfigMap, numaNodes int) (*kubeletconfig.KubeletConfiguration, string) {
 	// Configure Topology Manager in Kubelet with policy.
 	newCfg := oldCfg.DeepCopy()
 	if newCfg.FeatureGates == nil {
 		newCfg.FeatureGates = make(map[string]bool)
```

Comment:
we need to enable the TopologyManager Alpha-level policy options to make the current code work. But didn't we decide in the KEP to start from beta?

Reply:
either works, but the best fix is to move the new option to beta.

Reply:
We shouldn't have to, but you still have the option listed as an alpha feature here:
https://github.com/kubernetes/kubernetes/pull/124148/files#diff-441ff1f99d4d01db0b4d2ea6b26faee508b87af24e6dbae93229126cd4198cbaR36

Reply:
Isn't this new option already in beta? see https://github.com/kubernetes/kubernetes/pull/124148#discussion_r1558954149